.travis.yml

```diff
@@ -2,20 +2,48 @@ sudo: required
 language: go
 go_import_path: github.com/open-policy-agent/gatekeeper
 go:
-- "1.11"
+- "1.12.x"
 
-deploy:
-  - provider: script
-    skip_cleanup: true
-    script: make travis-dev-deploy
-    on:
-      repo: open-policy-agent/gatekeeper
-      branch: master
-      tags: false
-  - provider: script
-    skip_cleanup: true
-    script: make travis-dev-release
-    on:
-      repo: open-policy-agent/gatekeeper
-      branch: master
-      tags: true
+services:
+- docker
+
+jobs:
+  include:
+    - stage: test
+      script: make
+    - stage: e2e
+      before_script:
+      - KIND_VERSION=0.4.0
+      - KUSTOMIZE_VERSION=3.1.0
```

Comment:
nit: can we make sure we are using the same version of kustomize
https://github.com/open-policy-agent/gatekeeper/blob/master/test/Dockerfile#L17